packages/UI/src/components/molecules/Alert/StyledAlert.ts

```diff
@@ -28,4 +28,10 @@ export const Container = styled.div<Styles>`
       `;
     }}
   }
+
+  > button {
```

Comment:
it's recommended to write it this way: `& > button {`

Reply:
Ok